kratos/input_output/vtk_output.cpp

```diff
@@ -964,6 +964,11 @@ void VtkOutput::WriteIntegrationVectorContainerVariable(
     const auto& r_process_info = mrModelPart.GetProcessInfo();
     std::vector<TVarType> tmp_result;
     rContainer.begin()->CalculateOnIntegrationPoints(rVariable, tmp_result, r_process_info);
+    
+    // if no results, return
+    if (tmp_result.size() == 0) {
+        return;
+    }
```

Comment:
in which case would one not have results? If an element does not write output for a variable?

I am mostly concerned how this works in MPI, when some partitions write this and some dont, do you know?